helix-term/src/commands.rs

```diff
@@ -2116,7 +2116,8 @@ fn global_search(cx: &mut Context) {
                 let picker = FilePicker::new(
                     all_matches,
                     current_path,
-                    move |cx, FileResult { path, line_num }, action| {
+                    move |cx, file_result, action| {
+                        let Some(FileResult { path, line_num }) = file_result else { return; };
```

Comment:
This is a minor style nit but we should follow the recently merged (https://github.com/rust-lang/rust/pull/107312) style guide guidance for formatting these `let..else`s: https://github.com/rust-lang/rust/blob/master/src/doc/style-guide/src/statements.md#single-line-let-else-statements. Rustfmt won't automatically fix or lint for this style yet but it would be good to adhere to it now so we don't have odd formatting failures when we upgrade rustfmt: `let <pattern> = <expr> else { return };`

```suggestion
                        let Some(FileResult { path, line_num }) = file_result else { return };
```